cwl_utils/format.py

```diff
@@ -0,0 +1,65 @@
+from typing import List, Optional, Set, Union
```

Comment:
This module needs a docstring. And maybe a more descriptive name, like `file_formats.py`?